doc/data/messages/r/redefined-builtin/details.rst

```diff
@@ -0,0 +1,23 @@
+The `allowed-redefined-builtins <https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#allowed-redefined-builtins>`_
```

Comment:
Can you use an internal link here? This ties the link to the actual site, whereas it should be possible to link using an internal reference.

Reply:
thanks, i've updated to use internal reference